.github/workflows/apt-packages.yaml

```diff
@@ -81,6 +81,7 @@ jobs:
         fi
 
     - name: Test Downgrade
+      if: matrix.image != 'ubuntu:23.04'
```

Comment:
I guess this is needed because there are no older packages for ubuntu:23.04?

Reply:
yes downgrade test requires the previous version to be packaged as well which is not true for ubuntu 23.04 we can remove this when we package 2.12.1